CHANGES.rst

```diff
@@ -13,6 +13,18 @@ Changes
   The meta data us useful, for example, for selecting tests with the
   nose attribute mechanism.
 
+- Added ``doctestcase.doctestfiles``
+
+  - Define multiple doctest files at once.
+
+  - Automatically assign test class members.  So rather than::
+
+      test_foo = doctestcase.doctestfile('foo.txt')
+
+    You can use::
+
+      doctestcase.doctestfiles('foo.txt')
```

Comment:
"I'd probably be happier with a class decorator, or a factory function" I'm not sure how a class decorator would help.

WRT factory function, you'd end up with something like:

    class MyTests:
       ...

    doctestfiles(MyTest, 'foo.txt', 'bar.txt', ...)

While I agree that's a little less magical, it seems kinda gross to split the class definition between a class statement and a function call outside the class statement.